src/Components/Artwork/Metadata.tsx

```diff
@@ -32,13 +33,25 @@ export class MetadataContainer extends React.Component<MetadataProps> {
             user &&
             user.lab_features &&
             user.lab_features.includes("New Artwork Brick")
-          return (
+
+          const detailsContent = (
             <div className={className}>
-              <DetailsBlock showSaleLine={extended} artwork={artwork} />
+              <DetailsBlock
+                includeLinks={!enableLabFeature}
+                showSaleLine={extended}
+                artwork={artwork}
+              />
               {!enableLabFeature &&
                 extended && <ContactBlock artwork={artwork} />}
             </div>
           )
+          return enableLabFeature ? (
+            <StyledTextLink href={artwork.href}>
+              {detailsContent}
+            </StyledTextLink>
```

Comment:
What happens if this is just `<Serif><a href='#'>...</a></Serif>`? I (think) StyledTextLink refers to our old look, before we implemented the new design system

Reply:
answered above, but I was using `StyledTextLink` b/c that seemed to do what I want, which is: no underlining or underlining on hover, and you get the nice affect of the text going darker when you do hover, which matches the existing behavior as well.

With just a plain `<a>` tag I _was_ getting all that extra cruft (and no color change on hover).

Reply:
Cool cool, thanks for explaining 👍